designer/client/src/components/graph/Graph.tsx

```diff
@@ -375,13 +377,14 @@ export class Graph extends React.Component<Props> {
         this.panAndZoom = new PanZoomPlugin(this.processGraphPaper, this.viewport);
 
         if (this.props.isFragment !== true && this.props.nodeSelectionEnabled) {
-            const { toggleSelection, resetSelection } = this.props;
+            const { toggleSelection, resetSelection, handleStatisticsEvent } = this.props;
             new RangeSelectPlugin(this.processGraphPaper, this.props.theme);
             this.processGraphPaper
                 .on(RangeSelectEvents.START, () => {
                     this.panAndZoom.toggle(false);
                 })
                 .on(RangeSelectEvents.STOP, () => {
+                    handleStatisticsEvent({ selector: EventTrackingSelector.RangeSelectNodes, event: EventTrackingType.Keyboard });
```

Comment:
Yeah, I added it like this because it's a combination of keyboard and mouse (shift/ctrl/command + mouse move), I forgot that double click is also an option here. I will try to add two events here:
- Range by double-click
- Range by combination mouse + keyboard

Reply:
Ok i updated it and add two separate events for a range select